### PR TITLE
formular 

### PR DESCRIPTION
**Pull request by dimanoid1972**
I clear the file

### CODE REVIEW

**Review by PiotrKalachyn — COMMENTED** [review]
It is fine. I would only suggest putting inputs inside labels, then you don't need IDs

**Comment by PiotrKalachyn** [comment]
Why is the pull request named Formular? Can you think of a more descriptive title that actually tells what is being changed?

File: formular.html [hunk 1]
```diff
@@ -10,25 +10,13 @@ <h1>Date Matcher</h1>
...
-       <li>
-        <p>
-        <label for="Date2"></label>
-        <input type="date" name="Date2" id="Date2">
-        </p>
-       </li>
-      </ul>
-     </fieldset>
-      <p>
+      <label for="Date1"></label>
```
Comment (PiotrKalachyn) [inline formular.html]: There's a technique to put an `unput` tag *inside* its `label` tag. That way we organically tell what the label labels and can drop IDs

Comment (dimanoid1972) [reply formular.html]: Do I have to change it with this technique? I didn't understand, is it true or I have to do changes and send to you?
And what is this technique?

Comment (PiotrKalachyn) [reply formular.html]: The technique without id is:
```
<label>
    Date 1: 
    <input type="date" name="date1">
</label>
```

this way a label is connected with an input organically, without *for* and *id*

Comment (PiotrKalachyn) [inline formular.html]: please add text to the label element

File: formular.html [hunk 2]
```diff
@@ -10,25 +10,13 @@ <h1>Date Matcher</h1>
...
+      <input type="date" name="Date1" id="Date1">
+      <label for="Date2"></label>
```
Comment (PiotrKalachyn) [inline formular.html]: please add text to the label element

**Review by PiotrKalachyn — APPROVED** [review]
As a matter of taste, I would prefer the laconic "Date 1" to "Add a date1:", and definitely with a space between "date" and "1", as in human English. But technically all is fine now. So change that or go ahead like this, it is up to you.

**Comment by dimanoid1972** [comment]
Ok!

**Pull request merged**